packages/backend/src/helpers/flow-templates.ts

```diff
@@ -100,25 +101,39 @@ function replacePlaceholdersInParameters(
         placeholderReplacementMap,
       )
     } else if (typeof value === 'string') {
-      // Replace the value (placeholder) if it matches the regex
-      if (!PLACEHOLDER_REGEX.test(value)) {
-        continue
-      }
+      parameters[key] = replacePlaceholderValue(
+        value,
+        placeholderReplacementMap,
+      )
+    }
+  }
+  return parameters
+}
+
+function replacePlaceholderValue(
+  value: string,
+  placeholderReplacementMap: IJSONObject,
+): string {
+  const matches = value.match(PLACEHOLDER_REGEX) // need to replace every match
```

Comment:
suggestion: 

use a capture group in ur regex i.e. const PLACEHOLDER_REGEX = /<<([a-zA-Z0-9_.? ]+)>>/g
```
you could use value.replace(PLACEHOLDER_REGEX, (match, capture) => {
      return ...
})
```